forge_tests/helpers/TestBaseWorkflow.sol

```diff
@@ -1,385 +1,318 @@
+  // JBFundingCycleStore
+  JBFundingCycleStore internal _jbFundingCycleStore;
+  // JBToken
+  JBToken internal _jbToken;
+  // JBTokenStore
+  JBTokenStore internal _jbTokenStore;
+  // JBSplitsStore
+  JBSplitsStore internal _jbSplitsStore;
+
+  // JBController3_1(s)
+  JBController3_1 internal _jbController;
+
+  JBFundAccessConstraintsStore internal _jbFundAccessConstraintsStore;
+
+  // JBETHPaymentTerminalStore
+  JBSingleTokenPaymentTerminalStore3_1_1 internal _jbPaymentTerminalStore3_1_1;
+
+  // JBETHPaymentTerminal3_1_2
+  JBETHPaymentTerminal3_1_2 internal _jbETHPaymentTerminal3_1_2;
+
+  // JBERC20PaymentTerminal3_1_2
+  JBERC20PaymentTerminal3_1_2 internal _jbERC20PaymentTerminal3_1_2;
+
+  // AccessJBLib
+  AccessJBLib internal _accessJBLib;
+
+  // Default funding cycle data
+  mapping(uint256 => JBFundingCycleConfiguration) public defaultCycles;
+  JBFundingCycleData _defaultData;
+  JBFundingCycleMetadata _defaultMetaData;
+  JBGroupedSplits[] _defaultSplits;
+  JBFundAccessConstraints[] _defaultFundAccess;
+
+  //*********************************************************************//
+  // ------------------------- internal views -------------------------- //
+  //*********************************************************************//
+
+  function multisig() internal view returns (address) {
+    return _multisig;
+  }
+
+  function jbOperatorStore() internal view returns (JBOperatorStore) {
+    return _jbOperatorStore;
+  }
+
+  function jbProjects() internal view returns (JBProjects) {
+    return _jbProjects;
+  }
+
+  function jbPrices() internal view returns (JBPrices) {
+    return _jbPrices;
+  }
+
+  function jbDirectory() internal view returns (JBDirectory) {
+    return _jbDirectory;
+  }
+
+  function jbFundingCycleStore() internal view returns (JBFundingCycleStore) {
+    return _jbFundingCycleStore;
+  }
+
+  function jbTokenStore() internal view returns (JBTokenStore) {
+    return _jbTokenStore;
+  }
+
+  function jbSplitsStore() internal view returns (JBSplitsStore) {
+    return _jbSplitsStore;
+  }
+
+  function jbController() internal view returns (JBController3_1) {
+    return _jbController;
+  }
+
+  function jbAccessConstraintStore() internal view returns (JBFundAccessConstraintsStore) {
+    return _jbFundAccessConstraintsStore;
+  }
+
+  function jbPaymentTerminalStore() internal view returns (JBSingleTokenPaymentTerminalStore3_1_1) {
+    return _jbPaymentTerminalStore3_1_1;
+  }
+
+  function jbETHPaymentTerminal() internal view returns (JBETHPaymentTerminal3_1_2) {
+    return _jbETHPaymentTerminal3_1_2;
+  }
+
+  function jbERC20PaymentTerminal() internal view returns (JBERC20PaymentTerminal3_1_2) {
+    return _jbERC20PaymentTerminal3_1_2;
+  }
+
+  function jbToken() internal view returns (JBToken) {
+    return _jbToken;
+  }
+
+  function jbLibraries() internal view returns (AccessJBLib) {
+    return _accessJBLib;
+  }
+
+  function getDefaultCycleConfig(
```

Comment:
Can't use this after access constraints pr is merged, maybe it's removed there but idr.